template_feed/content/Microsoft.CheckTemplate/Company.CheckTemplate.csproj

```diff
@@ -5,6 +5,7 @@
     <DevelopmentDependency>true</DevelopmentDependency>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <PackageTags>BuildCheck</PackageTags>
```

Comment:
should we add more tags here?
e.g. 
<PackageTags>BuildCheck;MSBuild;Build;MSBuildCheck;BuildValidation;DotnetBuildTools</PackageTags>

Reply:
How about `BuildCheck;MSBuild` as an initial set?

Reply:
Can we have one more "msbuildcheck" that corresponds to a short template name?